sensitivity-api/src/test/resources/SensitivityComputationParametersError.json

```diff
@@ -1,12 +0,0 @@
-{
-  "version" : "1.0",
```

Comment:
It is normal that you have delete this file?

Reply:
I found that it is not used.